code/model/modelread.cpp

```diff
@@ -90,7 +92,7 @@ static uint Global_checksum = 0;
 // FreeSpace 2 shipped at POF version 21.17
 // Descent: FreeSpace shipped at POF version 20.14
 // See also https://wiki.hard-light.net/index.php/POF_data_structure
-#define PM_LATEST_ALIGNED_VERSION	2201
+#define PM_LATEST_ALIGNED_VERSION	2300
```

Comment:
If you have the define here, why are you using the magic numbers in the version checks???

Reply:
Because this will change as the model number gets updated, I'll add a new define later during cleanup for `PM_FIRST_VERTLIM_VERSION`